core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-table-skeleton/dot-table-skeleton.component.ts

```diff
@@ -0,0 +1,37 @@
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, computed, input } from '@angular/core';
+
+import { SkeletonModule } from 'primeng/skeleton';
+import { TableModule } from 'primeng/table';
+
+/**
+ * @class DotTableSkeletonComponent
+ *
+ * Represents a table component with skeleton loading feature with dynamic columns and rows
+ */
+@Component({
+    selector: 'dot-table-skeleton',
+    standalone: true,
+    imports: [CommonModule, SkeletonModule, TableModule],
+    templateUrl: './dot-table-skeleton.component.html',
+    styleUrl: './dot-table-skeleton.component.scss',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class DotTableSkeletonComponent {
+    /**
+     * Represents the name of the header of the table
+     *
+     */
+    columns = input.required<string[]>();
+    /**
+     * Represents the number of rows needed in the table with skeleton.
+     *
+     * @param {number} rows - The number of rows for the input.
+     */
+    rows = input<number>(5);
```

Comment:
@oidacra According to the conversations in Slack, all signals should start with $.